puppet/modules/slave/manifests/rvm.pp

```diff
@@ -17,32 +17,52 @@
   Class['Rvm::System']
 
   if $facts['rvm_installed'] {
+    if $facts['os']['release']['major'] == '9' {
+      exec { 'install openssl 1.1 through rvm':
+        command  => "rvm pkg install openssl",
+        creates  => "/usr/local/rvm/usr/lib/libssl.so",
```

Comment:
@evgeni How's this for being more specific?

Reply:
splendid!